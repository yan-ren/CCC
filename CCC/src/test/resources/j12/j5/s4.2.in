3
1 3 2
3
2 1 3
4
4 1 3 2
4
2 4 1 3
0
