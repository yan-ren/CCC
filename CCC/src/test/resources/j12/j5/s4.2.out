12
4
18
12
